lib/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegend.scss

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../../scss-shared/nx-container-helpers';
+@use '../../scss-shared/nx-text-helpers';
+
+.nx-threat-indicator-legend {
+  $max-grid-columns: 6;
+
+  display: grid;
+  gap: var(--nx-spacing-2x);
+  grid-template-columns: repeat($max-grid-columns, auto);
+  margin: var(--nx-spacing-6x) 0;
+  max-width: max-content;
+
+  .nx-threat-indicator-legend-header {
+    @include nx-text-helpers.bold;
+    color: var(--nx-color-text-dark);
+    grid-column: span $max-grid-columns;
+  }
+
+  .nx-threat-container {
```

Comment:
I think this class name needs to be more specific, probably a BEM element under `.nx-threat-indicator-legend`

Reply:
> I think this class name needs to be more specific, probably a BEM element under `.nx-threat-indicator-legend`

Changed to `nx-threat-indicator-legend__threat-container`:
1a4c6bee156c3bb6149a6ab31217e58137ad8b1c